portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/journal/webcontentdisplay/WebContentDisplay.testcase

```diff
@@ -584,6 +584,58 @@ Welcome to Liferay Community Edition Portal 7.4.0 CE GA1''';
 		WebContent.viewPGViaWCD(webContentContent = "WC WebContent Content");
 	}
 
+	@description = "This is a test for LPS-129438. View the changes of Border Styles in WCD on widget page."
+	@priority = "4"
+	test ConfigureLookAndFeelBorderStyleOfWCD {
+		task ("Assert the default value in border style on WCD") {
+			Navigator.gotoSitePage(
+				pageName = "Web Content Display Page",
+				siteName = "Test Site Name");
+
+			AssertCssValue(
+				locator1 = "Portlet#PORTLET_CONTENT",
+				locator2 = "border-color",
+				value1 = "rgba(0, 0, 0, 0)");
+
+			AssertCssValue(
+				locator1 = "Portlet#PORTLET_CONTENT",
+				locator2 = "border-width",
+				value1 = "1px");
+		}
+
+		task ("Open Look and Feel configuration on WCD") {
+			Portlet.gotoPortletOptions(
+				portletName = "Web Content Display",
+				portletOption = "Look and Feel Configuration");
+		}
+
+		task ("Configure Border Style tab") {
+			IFrame.selectConfigurationFrame();
+
+			Portlet.configureCustomBorderStylesViaLookAnfeelPG(
+				borderColorAttributeList = "Top-FF0D0D",
+				borderWidthAttributeList = "WidthTop-15",
+				borderWidthSelectList = "WidthTop-px",
+				checkToggleSwitchList = "Border Width,Border Color");
+
+			IFrame.closeFrame();
+
+			Refresh();
+		}
+
+		task ("Assert the new border style is applied to WCD") {
+			AssertCssValue(
+				locator1 = "Portlet#PORTLET_CONTENT",
+				locator2 = "border-color",
```

Comment:
View border-top-color, border-left-color, border-right-color and border-bottom-color